Enhance `useCreateListingModal` documentation

mithatakbulut opened the pull request:
- Added a note to the documentation highlighting the use of the `useBalanceOfCollectible` hook for verifying ownership before displaying the sell modal.
- Updated examples to utilize `marketplaceConfig` for dynamic collection data, improving clarity and usability.
- Refined code formatting and variable naming for consistency and readability.
- Enhanced error handling messages for better debugging experience.

ScreamingHawk commented on docs/pages/solutions/technical-references/internals/v1/guest-module.mdx:
```diff
@@ -1,2 +1,13 @@
+
+## On Demand deployment
+
+When a user interacts with their wallet on chain for the first time, the transaction will include the deployment of the wallet contract.
```
Found a few other instances of "on chain". Have updated in this PR as well

ScreamingHawk commented on docs/pages/solutions/technical-references/internals/v1/guest-module.mdx:
```diff
@@ -1,2 +1,13 @@
-# GuestModule and on-demand deployment
+# Guest Module
 
+The Guest Module is an implementation of the Sequence wallet that does not signatures, nonces or replay protection.
```
Using "does not support"

moskalyk commented:
looks good to me

mithatakbulut committed:
Enhance `useCreateListingModal` documentation

- Added a note to the documentation highlighting the use of the `useBalanceOfCollectible` hook for verifying ownership before displaying the sell modal.
- Updated examples to utilize `marketplaceConfig` for dynamic collection data, improving clarity and usability.
- Refined code formatting and variable naming for consistency and readability.
- Enhanced error handling messages for better debugging experience.

mithatakbulut committed:
Merge feb14257c15dc1fbf1dab7933cf686048643649d into ceba0ff45791bc8c05fe0ac0b3c22fa6c1d3e5c5

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 160/merge

mithatakbulut committed:
fix note copy

mithatakbulut committed:
Merge 285750075d8a4e0dd5f27bf6c292f89772ed58e6 into ceba0ff45791bc8c05fe0ac0b3c22fa6c1d3e5c5

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 160/merge
